homeassistant/components/russound_rnet/media_player.py

```diff
@@ -85,17 +86,22 @@ def __init__(self, hass, russ, sources, zone_id, extra):
         self._attr_name = extra["name"]
         self._russ = russ
         self._attr_source_list = sources
-        self._zone_id = zone_id
+        self._controller_id = math.ceil(zone_id / 6)
```

Comment:
Can we add an explanation here why our zone id is an ever growing number but the `russound` library expects it to be split between controller_id and zone_id?